src/System.Memory/ref/System.Memory.cs

```diff
@@ -246,6 +270,48 @@ public abstract partial class OwnedMemory<T> : System.Buffers.IRetainable, Syste
         public abstract void Retain();
         protected internal abstract bool TryGetArray(out System.ArraySegment<T> arraySegment);
     }
+    public static partial class ReadOnlyBufferExtensions
+    {
+        public static void CopyTo<T>(this System.Buffers.ReadOnlyBuffer<T> buffer, System.Span<T> destination) { }
+        public static System.Nullable<System.SequencePosition> PositionOf<T>(this System.Buffers.ReadOnlyBuffer<T> buffer, T value) where T : System.IEquatable<T> { throw null; }
+        public static T[] ToArray<T>(this System.Buffers.ReadOnlyBuffer<T> buffer) { throw null; }
+    }
+    public readonly partial struct ReadOnlyBuffer<T>
+    {
+        private readonly object _dummy;
+        public static readonly System.Buffers.ReadOnlyBuffer<T> Empty;
+        public ReadOnlyBuffer(System.Buffers.IMemoryList<T> startSegment, int startIndex, System.Buffers.IMemoryList<T> endSegment, int endIndex) { throw null; }
+        public ReadOnlyBuffer(System.Collections.Generic.IEnumerable<System.Memory<T>> buffers) { throw null; }
```

Comment:
thought we were nuking this ctor. /cc @KrzysztofCwalina ?

Reply:
I have no problem with that.